tests/brevitas/export/test_onnx_fp8.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2024, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from packaging import version
+import pytest
+import torch
+
+from brevitas import torch_version
+from brevitas.export import export_onnx_qcdq
+import brevitas.nn as qnn
+from brevitas.quant.experimental.float_quant_ocp import Fp8e4m3OCPWeightPerTensorFloat
+
+
+def test_simple_fp8_export():
+    if torch_version < version.parse('2.1.0'):
+        pytest.skip(f"OCP FP8 types not supported by {torch_version}")
+
+    model = qnn.QuantLinear(3, 16, weight_quant=Fp8e4m3OCPWeightPerTensorFloat)
+    export_onnx_qcdq(model, torch.randn(1, 3), 'test.onnx', export_weight_q_node=True)
```

Comment:
I guess we can extend this small test to run and compare results between pytorch and ONNX? Nothing too complicated, I guess at some point we would like to extend the export tests we already have